File: app/src/main/java/com/vpulse/ftpnext/commons/interfaces/OnStartAnimation.java
package com.vpulse.ftpnext.commons.interfaces;

import android.view.animation.Animation;

public abstract class OnStartAnimation implements Animation.AnimationListener {

    public abstract void onAnimationStart(Animation animation);

    @Override
    public void onAnimationEnd(Animation animation) {

    }

    @Override
    public void onAnimationRepeat(Animation animation) {

    }
}

File: app/src/main/java/com/vpulse/ftpnext/commons/interfaces/OnStartAnimation.kt
package com.vpulse.ftpnext.commons.interfaces

import android.view.animation.Animation

abstract class OnStartAnimation : Animation.AnimationListener {
    abstract override fun onAnimationStart(animation: Animation)
    override fun onAnimationEnd(animation: Animation) {}
    override fun onAnimationRepeat(animation: Animation) {}
}